caveclient/skeletonservice.py

```diff
@@ -313,13 +419,18 @@ def get_skeleton(
         if not self.fc.l2cache.has_cache():
             raise NoL2CacheException("SkeletonClient requires an L2Cache.")
 
-        url = self.build_endpoint(
+        url = self._build_endpoint(
             root_id, datastack_name, skeleton_version, output_format
         )
 
         response = self.session.get(url)
         self.raise_for_status(response, log_warning=log_warning)
 
+        if verbose_level >= 1:
```

Comment:
this should use the logging infrastructure rather than print statements to give meta-control to the program calling this as to how they want to handle these logs.

Reply:
Good idea. Thanks.

Reply:
I'll make this change, but it doesn't appear to play nicely with Jupyter Notebook. This output doesn't appear unless I run the following code in the notebook every time I reload the kernel:

`import logging
logger = logging.getLogger()
logger.setLevel(logging.DEBUG)`

I'm trying to find a way to configure this by default in the jupyter files, but haven't found it yet. Do you have your Jupyter environment configured so that it shows logging output?

Reply:
if you use logger.debug then this is what you get.. if you do logger.warning it will produce a warning which is usually included by default.  It is then up to the client about how they want to configure their log levels.. the snippet you showed shows how you can do that.  I don't know how to make jupyter do that by default on every kernel you start.